import/build.gradle

```diff
@@ -61,6 +62,8 @@ dependencies {
     compile project(":dicom_util")
     compile project(":util")
 
+    compile "com.github.red1408:deid-redactor:233908ca"
```

Comment:
Dusan said that he preferred us embedding a precompiled .jar from the deid redactor project rather than creating a clone repo. Are you ok with that change? I.e. building a .jar file and hosting it in this repo, and then having gradle compile it here.

Reply:
Ok, changed to pre-built redactor jar.